README.md

```diff
@@ -125,19 +125,37 @@ MySQL password should be the same for initialSetupTests as openmrs password
     ```
     npm install
     ```
+   
+## Setting up the local instance for the test backend.
 
-## Run local instance for the test backend.
+1. Clone the repository and navigate
+    ```
+    git clone -b 3.x git@github.com:openmrs/openmrs-distro-referenceapplication.git
+    cd openmrs-distro-referenceapplication
+    ```
 
-1. Navigate to the docker folder
-    ``` 
-    cd docker
+2. Package the distribution and prepare the run
     ```
-2. Run the app
+    mvn clean package
+    ```
+
+### If the build fails at this point should be destroyed the associated volumes
+```
+cd run/docker
+docker compose down -v
+mvn clean package
+```
```

Comment:
```suggestion
* If the build fails, try deleting existing containers using (`docker compose down -v`)
```
cd run/docker
docker compose down -v
mvn clean package
```
```

Reply:
@jayasanka-sack We should checkout to the docker directory to delete the existing container. So the cd `run/docker `command should be there, right?

Reply:
* If the build fails, navigate to the `run/docker` directory and try deleting existing containers using `docker compose down -v`

Reply:
Or maybe your original sugesstion: 

* If the build fails at this point should be destroyed the associated volumes
```
cd run/docker
docker compose down -v
```